lib/howl/interactions/grep_search.moon

```diff
@@ -0,0 +1,85 @@
+-- Copyright 2012-2015 The Howl Developers
+-- License: MIT (see LICENSE.md at the top-level directory of the distribution)
+
+import interact from howl
+import highlight from howl.ui
+import Matcher from howl.util
+
+interact.register
+  name: 'select_match'
+  description: ''
+  handler: (opts) ->
```

Comment:
This function ends up being a little bit too long; it seems like there should be helper functions that could be extracted.